x-pack/plugins/security_solution/public/common/components/markdown_editor/markdown_link.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { EuiLink, EuiLinkAnchorProps } from '@elastic/eui';
+
+type MarkdownLinkProps = { disableLinks?: boolean } & EuiLinkAnchorProps;
+
+const MarkdownLinkComponent: React.FC<MarkdownLinkProps> = ({
+  disableLinks,
+  href,
+  target,
+  children,
+  ...props
+}) => (
+  <>
+    {disableLinks ? (
+      <span>{children}</span>
+    ) : (
+      <EuiLink
```

Comment:
Would you be willing to restore the functionality in (deleted file) `markdown/index.tsx` that rendered the tooltip shown in the screenshot below?

<img width="370" alt="link-tooltip" src="https://user-images.githubusercontent.com/4459398/96619413-66598e80-12c3-11eb-8671-33e044cac98c.png">

Reply:
Oh, I totally forgot. Yes, I will!